### DISCUSSION TITLE
Stress Tests

### DISCUSSION THREAD

**Discussion by Jeevesy** [quabug/EntitiesBT#180] [General]
What stress tests have you done on this?

Is there anywhere I can see a demo of it?

Cheers

**Comment by quabug** [comment]
Haven't done any of stress tests yet, and IMO it is not that important since there's not too many running behavior trees for most of games. And it usually require specific optimizations for those rare cases, like simulation-game.

Some simple samples could be imported by this package https://openupm.com/packages/com.quabug.entities-bt.samples/, but it will soon be moved into `Assets`.

**Comment by Jeevesy** [comment]
Thank you. I want to simulate lots of animals making decisions
